src/components/Facility/BedCapacity.tsx

```diff
@@ -74,7 +67,11 @@ export const BedCapacity = (props: BedCapacityProps) => {
         const existingData = capacityQuery.data?.results;
         // if all options are diabled
         if (existingData.length === BED_TYPES.length) {
-          return;
+          //showing an error message "can't add more bed types" instead of returning
+          const errors = initForm;
+          errors["bedType"] = "Cannot add more bed types";
```

Comment:
Instead of doing this, why not disable the add button on the page 🤔 (in FacilityBedCapacity).

Reply:
Its the submit button is also disabled at this stage.I added this error to ensure user can understand why can't he/she add more bed types.

Reply:
I meant disabling the button on facility page.

<img width="361" alt="image" src="https://github.com/user-attachments/assets/7e0f3ddd-929b-4c9e-9e71-65594a7f30a0">

Reply:
Got it.
will make those changes. and remove that unnecessary error state update